src/pages/Product/Product.js

```diff
@@ -1,28 +1,41 @@
-import React from "react";
-import "./Product.scss";
+import React, { useState } from 'react';
+import Color from './components/Color/Color';
+import Count from './components/Count/Count';
+import Review from './components/Review/Review';
+import ColorButton from './components/ColorButton/ColorButton';
+import './Product.scss';
 
 const Product = () => {
+  const [count, setCount] = useState(1);
+  const [golfBallImg, setGolfBallImg] = useState('/images/golf-ball-white.jpg');
+  const [color, setColor] = useState('white');
+
   const price = 300;
-  const totalPrice = price;
+  const totalPrice = price * count;
 
   return (
     <div className="product">
       <div className="productDetail">
         <div className="productDetailImg">
-          <img
-            src={`/images/golf-ball-.jpg`} // color 이름에 따라 다른 이미지 경로 넣기
-            alt={`golf-ball`}
+          <img src={golfBallImg} alt="golf-ball" />
```

Comment:
```suggestion
          <img src={`/images/golf-ball-${color}.jpg`} alt="golf-ball" />
```
해당 이미지는 위처럼 color state의 변화에 따라서 구성 될 수 있도록 구현해주세요!